test/unit/presenters/api/services_index_presenter_test.rb

```diff
@@ -19,6 +19,23 @@ def setup
     assert_equal 20, presenter.products.size
   end
 
+  test "deleted products should not be shown" do
```

Comment:
Nice to add a test even through this was already working :+1:

Reply:
Well, I wanted to confirm we had the expected behaviour there! 😄